webpack.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    entry: ['babel-polyfill', './parser.js'],
+    output: {
+        path: './bin',
```

Comment:
`./client` sounds a lot better. I'm going to remove the `/bin` folder from the repo but have the clientize script spit out the bundle into a `client` folder instead.